migrations/multitenant/0010-delete-cache-cache-notifier-on-delete.sql

```diff
@@ -0,0 +1,14 @@
+CREATE FUNCTION tenants_delete_notify_trigger ()
+    RETURNS TRIGGER
+AS $$
+BEGIN
+    PERFORM
+        pg_notify('tenants_update', '"' || OLD.id || '"');
```

Comment:
what is this used for?

Reply:
When a DELETE request to tenants is issued, we currently don't clear the cache across all storage-api services. 
Only on the instance that received the DELETE request.

By using this trigger on delete all containers will have a consistent view of the tenants cache